src/test/java/org/folio/rest/impl/PasswordResourceTest.java

```diff
@@ -112,11 +115,12 @@ public static void setUpClass(final TestContext context) throws Exception {
     DeploymentOptions restVerticleDeploymentOptions = new DeploymentOptions().setConfig(new JsonObject().put(HTTP_PORT, port));
     vertx.deployVerticle(RestVerticle.class.getName(), restVerticleDeploymentOptions, res -> {
       try {
-        tenantClient.postTenant(null, res2 -> {
+        TenantAttributes t = new TenantAttributes().withModuleTo("mod-password-validator-1.0.0");
+        tenantClient.postTenant(t, res2 -> {
           async.complete();
         });
       } catch (Exception e) {
-        e.printStackTrace();
+        logger.error(e.getMessage());
```

Comment:
Can you use asyncAssertSuccess? https://vertx.io/docs/vertx-unit/java/#_asynchronous_assertions

    vertx.deployVerticle(RestVerticle.class.getName(), restVerticleDeploymentOptions, context.asyncAssertSuccess(id -> {
      TenantAttributes t = new TenantAttributes().withModuleTo("mod-password-validator-1.0.0");
      tenantClient.postTenant(t, context.asyncAssertSuccess());
    }));

* asyncAssertSuccess replaces async.
* asyncAssertSuccess replaces the logger because any Exception is reported as test failure.

Reply:
I'd prefer to keep this code as is, not a deprecation, valid usage